aea/configurations/schemas/aea-config_schema.json

```diff
@@ -7,7 +7,6 @@
     "aea_version",
     "agent_name",
     "author",
-    "description",
```

Comment:
Why are you removing the description?

Reply:
I don't see why that should be a mandatory field

Reply:
It is a mandatory field on registry; add it back please.